src/lib/components/PageHeader.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  export let testId: string | undefined = undefined;
+
+  let hasHeader: boolean;
+  $: hasHeader =
+    $$slots.topLeft !== undefined || $$slots.topRight !== undefined;
+</script>
+
+<div class="container" data-tid={testId}>
+  {#if hasHeader}
+    <div class="header">
+      <slot name="topLeft" />
+      <slot name="topRight" />
+    </div>
+  {/if}
+  <div class="content">
+    <slot name="title" />
+    <slot name="subTitle" />
+  </div>
+</div>
+
+<style lang="scss">
+  .container {
+    display: flex;
+    flex-direction: column;
+    gap: var(--padding-4x);
+  }
+
+  .header {
+    display: flex;
+    flex-direction: row;
```

Comment:
Isn't `row` the default?

Reply:
yes, I wanted to be specific.

Reply:
It might be incorrect but I've got the feeling that the bundler does not clean the CSS of its default tags when build. Therefore I personally think it's best to not be "specific" with CSS as it makes the CSS heavier.

But maybe there is another valid reason? Happy to discuss it.

Reply:
No strong opinion.

Reply:
then let's remove it if you don't mind

Reply:
done